lib/ansible/modules/windows/win_audit_policy_system.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2017 Noah Sparks <nsparks@outlook.com>
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: win_audit_policy_system
+short_description: Used to make changes to the system wide Audit Policy.
+description:
+  - Used to make changes to the system wide Audit Policy.
+  - It is highly recommended to take a backup of the policies before adjusting them for the first time.
+  - See this page for in depth information U(https://technet.microsoft.com/en-us/library/cc766468.aspx).
+version_added: "2.5"
+author:
+  - Noah Sparks (@nwsparks)
+options:
+  category:
+    description:
+      - Single string value for the category you would like to adjust the policy on.
+      - Cannot be used with I(subcategory). You must define one or the other.
+      - Changing this setting causes all subcategories to be adjusted to the defined I(audit_type).
+  subcategory:
+    description:
+      - Single string value for the subcategory you would like to adjust the policy on.
+      - Cannot be used with I(category). You must define one or the other.
+  audit_type:
+    description:
+      - The type of event you would like to audit for.
+    choices: [ 'success', 'failure', 'success and failure', 'none' ]
```

Comment:
The more I think about this the more I think this should be a list type where you can specify some the options `success` and `failure`. You can use a combination of both (`success and failure`) or just an empty list (`none`) for the other options. We should also decide on a default, potentially `none` is the default if it isn't set.

Reply:
I think I did it this way to make the values match what auditpol outputs so I wouldn't have to do string manipulation. I can take a look though and see about changing it to a list.

I wonder if making it required instead of a default would be better here to force a selection. Otherwise, for a default, I think 'success' would make the most sense. I think people using this module would typically looking to be enable auditing rather than disable. Either way though you're right, it does need one of these. I'll give some though about it...I guess there isn't much difference between the two really.

Reply:
Sounds good to me